react-commenting.js

```diff
@@ -30,19 +30,18 @@ class Board extends Component{
   }
 
   gamePlay = (index) => {
-    // 3) Here:
-    const { gameBoard, currentPlayer, winner, clickCount } = this.state
-    // 4) Here:
+    // 3) Here: This allows us to run the class constructor this.state in our function gamePlay
+    const { gameBoard, currentPlayer, winner } = this.state
```

Comment:
I think I know what you are getting at here, but this line in particular is destructuring the values out of state so we can call just the variable in the method without calling this.state.variable